### PR TITLE
Review backend new

### PR DESCRIPTION
**Pull request by YesratRahman**

### CODE REVIEW

File: findmyparadise_frontend/package.json [hunk 1]
```diff
@@ -33,6 +33,7 @@
     "prettier": "^2.2.1",
     "sass": "^1.26.11",
     "sass-loader": "^10.0.2",
-    "vue-cli-plugin-bootstrap-vue": "~0.7.0"
+    "vue-cli-plugin-bootstrap-vue": "~0.7.0",
+    "vue-cli-plugin-vuetify": "~2.3.1"
```
Comment (zmiles17) [inline findmyparadise_frontend/package.json:37]: Are we using Vuetify?

File: findmyparadise_frontend/src/components/Guests.vue [hunk 2]
```diff
@@ -1,13 +1,25 @@
 <template>
   <div>
     <h5>
-      Adults: <button type="button" id="DecresaeAdults" class="btn" @click="DecreaseAdults()">-</button> {{AdultsNum}} <button type="button" class="btn" @click="IncreaseAdults()">+</button>
+      Adults:
+      <button
+        type="button"
+        id="DecresaeAdults"
```
Comment (zmiles17) [inline findmyparadise_frontend/src/components/Guests.vue:7]: typo

File: findmyparadise_frontend/src/components/Guests.vue [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+        class="btn"
+        @click="DecreaseAdults()"
```
Comment (zmiles17) [inline findmyparadise_frontend/src/components/Guests.vue:9]: Let's all agree to camel case methods and variables. decreaseAdults(), increaseAdults(), adultsNum, etc

**Review by zmiles17 — COMMENTED** [review]
I realized after commenting that you hadn't worked on that components that I commented on.

**Review by tamera-brown — APPROVED** [review]

**Pull request merged**